### PR TITLE
Add translation request templates

### LINKED ISSUE #4835: Create template to submit new full translation requests
This issue will be used to submit new full translation requests.

Our users will be able to know the progress of these translations, based on this issue, the corresponding PR and its position on the translation project (or wherever else we might decide to keep these issues and PRs)

### LINKED ISSUE #4836: Create a manual PR template (to copy and paste) to submit full translation PRs
Each translation should have one issue that starts it (format specified on #4835) and one PR with the format specified on this issue. (To be included in the docs folder or somewhere else, maybe a translations folder).

The PR template should contain a checklist with some tasks to be done, both on android & iOS, including some actual testing.

The checklist should make sure file is in the right format, check usage of whitespaces, there's no contents in the entries comments, follows the guide defined on #4834, and maybe something more.

The PR should also make sure that the right people are notified and the translation has permission to enter our code base.

**The PR template is only a file that will exist in our docs or translations folder and will be copy/pasted to create the PR for the translations**

### PR DESCRIPTION
**Pull request by daividssilverio**
## What's this?
This PR adds the issue and pull request templates for new translation requests.

### Relationships
Closes #4835 
Closes #4836 

## Why do we want this?
To be organized when adding new translations to our codebase.

## How is it done?
It just adds two new files.
One to be used when a new translation is to be requested either by us or clients themselves.
Another to be used as a manual (or not) pull request template when the translation is done and needs to be merged into our codebase.

### Why not another way?
Feel free to make suggestions, improve wording, anything that could improve the experience for new translations.

### Side effects
There shouldn't be any.

## Review hints
Check the .md files.

## :squid: Permissions
🦑 i18n team + This needs a look from @maritoggl and/or @amulware. No hurry in merging these.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/g-translation-request.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+## Target Language: <!-- Requested Language Here -->
+<!-- Please check that no other request for the same language has been made before submitting this issue. Leave a comment in the issue if you'd like collaborate -->
+
+## Requested at: <!-- enter date of request -->
+
+## Progress
+
+- [ ] Translations started
```
Comment (heytherewill) [inline .github/ISSUE_TEMPLATE/g-translation-request.md]: I think you can safely drop the word `Translation` from every step 😄

File: .github/PULL_REQUEST_TEMPLATE/translation-pull-request-template.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
+## Translation for <!-- Language Here -->
+
+Request issue: #TranslationIssueNumber
+
+## Status
+Don't consider this PR ready to be merged before the following steps are done:
+
+- [ ] The translator has been provided with a test build (or screenshots), run and approved how it looks (Ideal, but maybe not realistic)
```
Comment (heytherewill) [inline .github/PULL_REQUEST_TEMPLATE/translation-pull-request-template.md]: You could replace `(Ideal, but maybe not realistic)` with the less pessimistic `(Optional)`

File: .github/PULL_REQUEST_TEMPLATE/translation-pull-request-template.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+- [ ] This PR branch has been tested and looking good on multiple devices sizes with different font/zoom configurations (At least run the whole app in the smallest & highest supported versions of iOS and android)
```
Comment (heytherewill) [inline .github/PULL_REQUEST_TEMPLATE/translation-pull-request-template.md]:
```suggestion
- [ ] This PR's branch has been tested and looks good on multiple devices sizes with different font/zoom configurations (The bare minimum to verify this is running the whole app in the smallest & highest supported versions of iOS and Android)
```

File: .github/PULL_REQUEST_TEMPLATE/translation-pull-request-template.md [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+- [ ] Every text entry has been translated (Manually verified or generic Unit tests for translated text entries)
```
Comment (heytherewill) [inline .github/PULL_REQUEST_TEMPLATE/translation-pull-request-template.md]: I think you should remove the comment in parenthesis while we don't have the unit tests

File: .github/ISSUE_TEMPLATE/g-translation-request.md (hunk 1 above)
Comment (daividssilverio) [reply .github/ISSUE_TEMPLATE/g-translation-request.md]: I've changed it o/

**Pull request merged**